apps/graphql/src/apps/itinerary/dataloaders/Itineraries.js

```diff
@@ -71,20 +77,59 @@ const addReturnSearchQueryParams = inboundDate => {
   };
 };
 
+const parseSlugsToIds = async (
```

Comment:
Also, just realising that this should be done in a locations dataloader, so we can optimise this data-fetching. 
Now we have no caching for fetching these locations, and we could end up hitting the rest-api again when it is not necessary.